### ISSUE TITLE
Feature matching loss

### ISSUE THREAD

**Issue by mmehdig** [crux82/ganbert-pytorch#3]
It seems there is a tiny mismatch between the paper formula and the code on feature matching loss:

This is not exactly an L2-norm:

```
g_feat_reg = torch.mean(torch.pow(torch.mean(D_real_features, dim=0) - torch.mean(D_fake_features, dim=0), 2))
```

It should be like:

```
g_feat_reg = torch.sum(torch.pow(torch.mean(D_real_features, dim=0) - torch.mean(D_fake_features, dim=0), 2))
```

But I guess it doesn't change the outcome other than affecting the learning rates.

**Comment by crux82** [comment]
I totally agree with you. 

In my early experiments, it had no real impact on the results. However, I still need to confirm this in more experimental evaluations. 

Thank you for your comment!

Bests

Danilo
